### DISCUSSION TITLE
InstantID-hf likes to eat VRAM 2112x2112 37GB VRAM needed more than this Res crush at A100

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [lllyasviel/Fooocus#2682] [General]
![image](https://github.com/lllyasviel/Fooocus/assets/42430868/911bc0cf-a4d1-4a68-bb9a-775be44671df)
2
![nice gilmour 2](https://github.com/lllyasviel/Fooocus/assets/42430868/68de33d7-d4f9-4e1e-b356-ef570cda0bed)

**Comment by mashb1t** [comment]
InstantID is known for using much resources. As this has nothing to do with Fooocus i'd suggest you open a discussion in the InstantID repository.
